log4j-core-test/pom.xml

```diff
@@ -379,6 +379,22 @@
           </annotationProcessorPaths>
         </configuration>
       </plugin>
+      <!--
+      https://junit-pioneer.org/docs/environment-variables/#warnings-for-reflective-access
+      -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>
+            <!-- for environment variable access -->
+            --add-opens java.base/java.util=ALL-UNNAMED
+            --add-opens java.base/java.lang=ALL-UNNAMED
+            <!-- for URLStreamHandlerFactory access -->
+            --add-opens java.base/java.net=ALL-UNNAMED
```

Comment:
We have some tests that try to modify internals of `URLStreamHandlerFactory` to test something related to handling `URL`s. Yet another sign we could use our own `Resource`-style API. :)

Reply:
Also seems like the tests using this are related to `ResolverUtil` which is going away in 3.x as we no longer do class path scanning.